noir-projects/noir-protocol-circuits/crates/types/src/address/aztec_address.nr

```diff
@@ -52,16 +66,55 @@ impl Deserialize<AZTEC_ADDRESS_LENGTH> for AztecAddress {
     }
 }
 
+impl ToPoint for AztecAddress {
+    fn to_point(self) -> Point {
+        // Calculate y^2 = x^3 - 17
+        let y_squared = pow(self.inner, 3) - 17;
```

Comment:
I'd do `let x = self.inner` and mention that this is how an address is computed.